### PR TITLE
refactoring and type fix

### PR DESCRIPTION
**Pull request by beliaev-maksim**
Hello,
just typo fix and couple of small refactors

skip news
skip issue

### CODE REVIEW

**Review by serverwentdown — APPROVED** [review]
Corrects minor typos for slightly better consistency and one spelling mistake.

File: Lib/unittest/test/test_assertions.py [hunk 1]
```diff
@@ -106,7 +106,6 @@ class A:
         wr = None
 
         class Foo(unittest.TestCase):
-
```
Comment (serverwentdown) [inline Lib/unittest/test/test_assertions.py:109]: Just read the devguide, I think this would be rejected as it is a [formatting change](https://devguide.python.org/pullrequest/?highlight=formatting#making-good-prs).

Also mentioned [here](https://devguide.python.org/pullrequest/?highlight=cosmetic#making-good-commits) is "Do not do cosmetic changes to unrelated code in the same commit as some feature/bugfix".

**Review by serverwentdown — CHANGES_REQUESTED** [review]

File: Lib/unittest/test/test_assertions.py (hunk 1 above)
Comment (iritkatriel) [reply Lib/unittest/test/test_assertions.py:109]: That's correct.
